logstore/logstore.go

```diff
@@ -186,15 +188,18 @@ func (ls *logstore) AddLog(id thread.ID, lg thread.LogInfo) error {
 	ls.Lock()
 	defer ls.Unlock()
 
-	err := ls.AddPubKey(id, lg.ID, lg.PubKey)
-	if err != nil {
-		return err
-	}
 	if lg.PrivKey != nil {
-		if err = ls.AddPrivKey(id, lg.ID, lg.PrivKey); err != nil {
+		if pk, _ := ls.PrivKey(id, lg.ID); pk != nil {
```

Comment:
There can only be one "owned" log, with the private key. Fail here if we're trying to add a new PrivateKey. The alternative would be to not fail, and overwrite, but that seems dangerous.